app/helpers/date_time_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module DateTimeHelper
+  def format_time(datetime)
+    datetime.strftime("%-l:%M%P")
+  end
+
+  def scheduled_date_placeholder
+    publish_time ||= Time.current.tomorrow.change(hour: 9)
```

Comment:
Helpers are usually generic, so this should probably be renamed to something like `placeholder_time`.